app/views/medications/_form.html.erb

```diff
@@ -62,6 +62,21 @@
           </div>
         </div>
 
+        <div class="field">
+          <div class="label">
+            <%= f.label t('medications.form.weekly_dosage'), for: 'medication_weekly_dosage' %>
+              <%= content_tag(:span, '<i class="fa fa-question-circle"></i>'.html_safe, class: 'yes_title medication_tooltip_padding', title: t('medications.form.weekly_dosage_hint')) %>
+              <i class="fa fa-asterisk"></i>
+            <div class="clear"></div>
+          </div>
+          <div class="inline-block">
+            <% (0..6).each do |day| %>
```

Comment:
Might be helpful to stick this in a variable just to give context what 0..6 means.